.github/workflows/chaos.yml

```diff
@@ -11,22 +11,20 @@ on:
         type: string
 
 jobs:
-  Chaos_Testing_Build_Contracts:
+  chaos_testing_build_contracts:
+    name: Chaos Testing Build Contracts
     runs-on: ubuntu-latest
     container:
       image: projectserum/build:v0.22.1
       env:
         RUSTUP_HOME: "/root/.rustup"
         FORCE_COLOR: 1
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
     steps:
       - uses: actions/checkout@v2
       - run: echo $HOME
       - run: echo $PATH
       - name: Set Npm Config
```

Comment:
Remove this name here, as other `yarn install` steps don't have an explicit name.